added examples for editing privileges (#3986)
mattermost/docs: source/deployment/admin-roles.rst

Assigning Admin Roles
---------------------

> System Admins can assign roles using the mmctl tool. This can be done either locally or remotely.

Reviewer: ```suggestion
System Admins can assign roles using the mmctl tool, either locally or remotely.
```

``mmctl permissions role assign system_user_manager bob-smith sue-clark``

**To grant the Read Only Admin role to two users, Bob Smith and Sue Clark:**

``mmctl permissions role assign system_read_only_admin bob-smith sue-clark``

Editing Privileges of Admin Roles (Advanced)
--------------------------------------------

> Each of the admin roles have defined, default privileges as outlined above. However both read and write access to areas of the System Console can be granted or removed from each role by System Admins using the mmctl tool. This can be done either locally or remotely.

Reviewer: Does this mean that although there are default settings, those defaults can be modified? For example, can the user manager role be given more access over and above the default access? Can the System Manager have some access removed? If so, is there a table indicating which defaults have been modified?

Author: Yes, that is what that means. No, there is not currently a way to see what has been changed from defaults,  but the UI (coming in a future release) will eventually reflect that. We are also planning to add a "reset to defaults" switch.

Reviewer: Awesome, thank you!

The format of the mmctl command is:

``mmctl permissions add [role_name] [permission...]``

**To grant write access to the authentication section for all users with the User Manager role:**

``mmctl permissions add system_user_manager sysconsole_write_authentication``

**To grant read only access to the authentication section for all users with the User Manager role:**

``mmctl permissions remove system_user_manager sysconsole_read_authentication``

**To remove write access to the authentication section for all users with the User Manager role:**

``mmctl permissions remove system_user_manager sysconsole_write_authentication``

> Frequently Asked Questions

Reviewer: Can we add: "How do I remove a user from a role?"

Author: yes. added under `Assigning Admin Roles`

``mmctl permissions role assign system_user_manager bob-smith sue-clark``

**To grant the Read Only Admin role to two users, Bob Smith and Sue Clark:**

``mmctl permissions role assign system_read_only_admin bob-smith sue-clark``

**To remove the System Manager role from a single user called Bob Smith:**

``mmctl permissions role unassign system_manager bob-smith``

Editing Privileges of Admin Roles (Advanced)
--------------------------------------------

> Each of the admin roles have defined, default privileges as outlined above. However both read and write access to areas of the System Console can be granted or removed from each role by System Admins using the mmctl tool. This can be done either locally or remotely.

Reviewer: ```suggestion
Each of the admin roles have defined, default privileges as outlined above. 

System Admins can grant read and write access to other areas of the System Console, as well as remove read write access (including default access), for each role. This is done via the mmctl tool, either locally or remotely.
```

The format of the mmctl command is:

``mmctl permissions add [role_name] [permission...]``

> **To grant write access to the authentication section for all users with the User Manager role:**

Reviewer: ```suggestion
**To grant write access to the Authentication area of the System Console for all users with the User Manager role:**
```

``mmctl permissions add system_user_manager sysconsole_write_authentication``

> **To grant read only access to the authentication section for all users with the User Manager role:**

Reviewer: ```suggestion
**To grant read only access to the Authentication area of the System Console for all users with the User Manager role:**
```
